### PR TITLE
Improve context menu in manager

### PR DESCRIPTION
**Pull request by Bouh**
Main PR #1204 

Sorry for this but i've deleted my fork and all my PR are now broken with 
Bouh wants to merge 2 commits into 4ian:master from **unknown repository**
**unknown repository** arggg, i can't reclaim i guess it so i reopen this here...

### CODE REVIEW

File: newIDE/app/src/ProjectManager/index.js [hunk 1]
```diff
@@ -882,6 +939,8 @@ export default class ProjectManager extends React.Component<Props, State> {
                         eventsFunctionsExtension
                       )
                     }
+                    addLabel={'Add a New Extension'}
+                    onAdd={() => this._addEventsFunctionsExtension(name, i)}
```
Comment (4ian) [inline newIDE/app/src/ProjectManager/index.js]: Just thinking about it now, `name` arguments are actually not used at all right? Can you remove them? :)

File: newIDE/app/src/ProjectManager/index.js [hunk 2]
```diff
@@ -345,6 +352,50 @@ export default class ProjectManager extends React.Component<Props, State> {
     this.forceUpdate();
   };
 
+  _addLayout = (name: string, index: number) => {
```
Comment (4ian) [inline newIDE/app/src/ProjectManager/index.js]: name is not used?

File: newIDE/app/src/ProjectManager/index.js (hunk 1 above)
Comment (Bouh) [reply newIDE/app/src/ProjectManager/index.js]: yes

**Comment by 4ian** [comment]
Looks good!

**Pull request merged**